docs/request.schema.json

```diff
@@ -59,10 +52,10 @@
             "type": "string",
             "description": "The HTTP method of the request",
             "meta:enum": {
-                "GET": "GET request",
-                "POST": "POST request",
-                "PUT": "PUT request (not handled by Helix)",
-                "DELETE": "DELETE request (not handled by Helix)"
+                "get": "GET request",
```

Comment:
I think we should allow both: `get` and `GET`....

Reply:
I just noticed that methods are always already lowercased, while the documentation had them uppercased...

@trieloff

Reply:
I'd say we should revert OpenWhisk's lowercasing, but in order to do it consistently, we would also need to apply it to request headers, where it is not as reversible. Let's allow both.

Reply:
Some more context: the `meta:enum` property in the schema is for documentation, not validation. Therefore, I'd keep it uppercase in the `meta:enum`, but mention in the `description` that the method can be case insensitive. 

If we want to be more prescriptive with the methods allowed, we could add a `pattern`  or `enum` validation property, but that would rule out more creative HTTP method names,

Reply:
ok, reverted to uppercase and updated description.